docs/_articles/en/einstein/einstein-setup.md

```diff
@@ -110,3 +118,18 @@ Create a scratch org using the `SFDX: Create a Default Scratch Org...` command r
 Run **Einstein: Show Prompt History** from the Command Palette to open the Feedback console. When opened, you can view a running history of your prompts and associated responses. Use 👍, 👎, and comments for each response to provide us with feedback about the quality of the generated code. Your feedback during this beta is key to helping us improve AI model quality and overall product.
 
 ![feedback panel](./images/einstein-feedback.png)
+
+## Keyboard Shortcuts for Einstein for Developers
+
+You can use the default keyboard shortcuts in Visual Studio Code when using Einstein for Developers. You can rebind the shortcuts in the Keyboard Shortcuts editor using your preferred keyboard shortcuts for each specific command. For more information see [Key Bindings for Visual Studio Code](https://code.visualstudio.com/docs/getstarted/keybindings).
+
+You can search for each keyboard shortcut by command name in the Keyboard Shortcuts editor.
+
+| Command                                  | Key Binding |
+| ---------------------------------------- | ----------- |
+| Accept                                   | ⌘Enter      |
+| Accept                                   | Tab         |
+| Clear                                    | Escape      |
+| Einstein:Generate Code                   | ⇧⌘R         |
+| SFDX: Insert Comment for Code Generation | ⌘L          |
```

Comment:
Comment to Code is behind a feature flag - I think it can be omitted from this list.